### ISSUE TITLE
[enhancement] Add artist and album browser column

### ISSUE THREAD

**Issue by mojo2012** [vincelwt/harmony#209]
Add something like this to make it easiert to browse by artist or album:
![bildschirmfoto 2018-03-15 um 07 17 57](https://user-images.githubusercontent.com/1330361/37447137-052bc8ec-2821-11e8-97a8-bd4ce3dd9c20.png)

**Comment by vincelwt** [comment]
Mmm, this is great and pretty useful but would bloat a little Harmony. The motto has always been to keep it clean and minimalist.
Also how to make it work with for example Soundcloud, Reddit or HypeMachine? (Where you don't have a library of tracks)
It could be done through a plugin tho.

**Comment by mojo2012** [comment]
I don't consider it bloat - it's an essential part of the navigation experience.
See it as an extension to the search/filter functionality.
You have the artist information at least in soundcloud and spotify. I think you have the necessary information in all plugins. Think of it as another way to structure the existing data in another way.
Of course this should be optional (toolbar switcher between list and column view)

**Comment by vincelwt** [comment]
You are right. I've added it to the [features requests](https://trello.com/b/aO4L9OyM/harmony-features-requests) board see if it would appeal to more users (I don't doubt)
It would require quite some significant changes tho, so this isn't priority (bugs and usability improvements are) right now.
